src/nightwatch/snapshot.js

```diff
@@ -0,0 +1,37 @@
+const Jimp = require('jimp');
+const fs = require('fs');
+const path = require('path');
+
+
+module.exports = function (browser, done) {
+  const testDirectory = path.dirname(module.parent.filename);
+  const currentTest = browser.currentTest;
+
+  browser.execute('document.documentElement.getAttribute("lang")', [], (locale) => {
+    const snapshotDirectory = path.join(testDirectory, '__snapshots__', locale.value || 'en');
+
+    browser.screenshot(false, (data) => {
+      Jimp.read(Buffer.from(data.value, 'base64')).then((snapshot) => {
+        const name = currentTest.name.replace(/\s+/g, '-');
```

Comment:
thinking of biting the bullet here and just importing a module to poperly sanitize file names and camelcase.

Reply:
We discussed trying for the test step number and then falling back on the testStep name. For either:

`[1] Displays Correctly`  to be `name = [1]` or `Displays Correctly` to be `Displays-Correctly`

Reply:
Yup, haven't done that yet